proteus/SplitOperator.py

```diff
@@ -150,6 +150,57 @@ def setFromOptions(self,soOptions):
 
 Sequential_FixedStep = SO_base
 
+class Sequential_tnList(SO_base):
```

Comment:
How does this differ from Sequential_FixedStep_Simple

Reply:
As I understand the logic, of `Sequential_FixedStep_Simple`, `self.dt_system` is first set based on the length of the initial time step, which works fine.  
For subsequent time steps, `self.dt_system` starts at the value used in the previous iteration.  Thus, in `Sequential_FixedStep_Simple`'s  `stepExact_system` function (which uses the version written in the `SO_base` base class, see - https://github.com/erdc/proteus/blob/master/proteus/SplitOperator.py#L57 ), if `self.dt_system` is not large enough to reach the next time step, `self.dt_system` is set to step half the distance to `tExact`.  
In the case of wanting to replicate `tnList` exactly, it seems to me that we would want to set `self.dt_system` to give the exact step.  Thus, the difference is that `Sequential_tnList` sets `self.dt_system` to arrive at the next element of `tnList` in all cases (see - https://github.com/erdc/proteus/blob/c3159a489aae5f15266010b53feb4e355bc26017/proteus/SplitOperator.py#L167 )

Reply:
Got it. I don't think there's any reason to have both versions, so I removed the definition of Sequential_FixedStep_Simple and made it an alias of Sequential_tnList so it doesn't break anything.